src/core/public/rendering/rendering_service.tsx

```diff
@@ -48,22 +50,24 @@ export class RenderingService {
 
     ReactDOM.render(
       <I18nProvider>
-        <>
-          {/* Fixed headers */}
-          {chromeHeader}
+        <CoreThemeProvider theme$={theme.theme$}>
```

Comment:
This is core's main usage of the theme provider, wrapping the whole chrome UI with `EuiThemeProvider`